doc/debug.md

```diff
@@ -2,18 +2,32 @@
 
 ## FPS counter
 
-Flame provides the `FPSCounter` mixin for recording the fps, this mixin can be applied on any class that extends from `Game`. When applied you have to implemented the `recordFps` method and it should return `true` if you want to access the current fps by using the `fps` method`.
+Flame provides the `FPSCounter` mixin for recording the fps, this mixin can be applied on any class
+that extends from `Game`. Once applied you can access the current fps by using the `fps` method,
+like shown in the example below.
 
 ```dart
-class MyGame extends Game with FPS {
+class MyGame extends Game with FPSCounter {
+  static final fpsTextConfig = TextConfig(color: BasicPalette.white.color);
+
   @override
-  bool recordFps() => true;
+  void render(Canvas canvas) {
+    super.render(canvas);
+    fpsTextConfig.render(canvas, fps(120).toString(), Vector2(0, 50));
+  }
 }
 ```
 
 ## BaseGame features
 
-Flame provides some features for debugging, these features are enabled when the `debugMode` from the `BaseGame` class is set to `true` (or overridden to be true).
-When it's enabled all `PositionComponent`s will be wrapped into a rectangle, and have its position rendered on the screen, so you can visually verify the component boundaries and position.
+On `BaseGame` you don't have to apply the `FPSCounter` mixin to get access to the current fps,
+because it is already applied by default on the `BaseGame` class, so you can use the `fps` method directly.
```

Comment:
```suggestion
If you are using `BaseGame`, you don't have to apply the `FPSCounter` mixin to get access to the current FPS,
because it is already applied by default (so you can use the `fps` method directly).
```